rfcs/text/0000-extend-file-pe.md

```diff
@@ -0,0 +1,1076 @@
+# 0000: Extend the PE subfield of the File fieldset
+
+- Stage: **1 (proposal)**
+- Date: **TBD**
+
+The Portable Executable (PE) sub-field, of the `file` top-level fieldset, can be updated to include more file attributes to aid in file analysis. This additional document metadata can be used for malware research, as well as coding and other application development efforts.
+
+## Fields
+
+**Stage 0**
+
+This RFC is to create 25 additional sub-fields within the `file.pe` fieldset.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| file.pe.authentihash | keyword | Authentihash of the PE file. |
+| file.pe.compile_timestamp | date | Compile timestamp of the PE file. |
+| file.pe.compiler_product_versions | keyword | Version of the compiler. |
```

Comment:
here's a sample of what is in there today, as returned by VT. I'm not a PE expert, so this can probably get cleaned up, but it presumably means something to somebody. @rw-access, @Samirbous  do you have any input here?

```json
        "compiler_product_versions": [
          "id: 259, version: 27412 count=18",
          "id: 253, version: 23601 count=2",
          "id: 260, version: 27412 count=36",
          "id: 261, version: 27412 count=208",
          "id: 262, version: 27412 count=1",
          "id: 257, version: 27412 count=35",
          "[---] Unmarked objects count=841",
          "id: 260, version: 28920 count=18",
          "id: 259, version: 28920 count=11",
          "id: 261, version: 28920 count=466",
          "id: 261, version: 29111 count=15",
          "id: 255, version: 29111 count=1",
          "id: 151, version: 0 count=1",
          "id: 258, version: 29111 count=1"
        ]
```

another, more meaningful example:

```json
        "compiler_product_versions": [
          "[IMP] VS2008 SP1 build 30729 count=3",
          "[---] Unmarked objects count=10",
          "id: 174, version: 40219 count=11",
          "[LNK] VS2010 SP1 build 40219 count=1"
        ]
```